repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTLauncher.java

```diff
@@ -28,18 +29,21 @@
  * This is the launcher class for Repairnator realtime.
  */
 public class RTLauncher {
+
     private static Logger LOGGER = LoggerFactory.getLogger(RTLauncher.class);
     private List<SerializerEngine> engines;
     private RepairnatorConfig config;
     private EndProcessNotifier endProcessNotifier;
     private TimedSummaryNotifier summaryNotifier;
 
+    private String activeMQUrl;
```

Comment:
those two fields should go in RepairnatorConfig instead

Reply:
Hi :) , those fields can be removed without any problem.